libbeat/template/load.go

```diff
@@ -135,8 +169,15 @@ func (l *Loader) Load() error {
 // LoadTemplate loads a template into Elasticsearch overwriting the existing
 // template if it exists. If you wish to not overwrite an existing template
 // then use CheckTemplate prior to calling this method.
-func (l *Loader) LoadTemplate(templateName string, template map[string]interface{}) error {
+func (l *Loader) LoadTemplate(templateName string, template common.MapStr) error {
 	logp.Debug("template", "Try loading template with name: %s", templateName)
+	if l.client == nil {
+		if _, err := os.Stdout.WriteString(template.StringToPrint() + "\n"); err != nil {
+			return fmt.Errorf("Error writing template: %v", err)
```

Comment:
No capitalization in error string.

Also, I see that we check in both places that client is set before doing an actual call. Is there any reason why the client is could not have been set before using that type?

Reply:
When exporting the template to `Stdout` with the `./beat export template` cmd no ES client would be set. Thus, this nil check is necessary to see if we should export to ES or to Stdout.

So far the whole template loading functionality was implemented twice, once here and once in the [export code](https://github.com/elastic/beats/pull/9247/files#diff-a9cf870563a8b85ebf1d9e39341b45bb). I refactored to reuse the same code for both.